float32 x_goal
float32 y_goal   #goal position x and y
---
bool success         # Did it achieve it?